app/helpers/traffic_helper.rb

```diff
@@ -6,10 +6,16 @@ module TrafficHelper
   PERIOD_LENGTH = 15 # minutes
   CACHE_FOR = 5 # minutes
 
+  # TODO: ISS-809 – temporary measure for migration performance issue.
```

Comment:
Just a head up, [rubocop fails](https://travis-ci.org/github/lobsters/lobsters/builds/763025683#L529) in CI
```
Offenses:
app/helpers/traffic_helper.rb:9:19: C: Style/AsciiComments: Use only ascii symbols in comments.

  # TODO: ISS-809 – temporary measure for migration performance issue.
                  ^
176 files inspected, 1 offense detected
The command "bundle exec rubocop" exited with 1.
```